sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -348,6 +349,44 @@ export interface FileParallelUploadOptions extends CommonOptions {
     umask?: string;
 }
 
+// @public
+export interface FileQueryCsvTextConfiguration extends FileQueryTextConfiguration {
+    columnSeparator?: string;
+    escapeCharacter?: string;
+    fieldQuote?: string;
+    hasHeaders?: boolean;
+    kind: "csv";
+    recordSeparator: string;
+}
+
+// @public
+export interface FileQueryError {
+    description: string;
+    isFatal: boolean;
+    name: string;
+    position: number;
+}
+
+// @public
+export interface FileQueryJsonTextConfiguration extends FileQueryTextConfiguration {
+    kind: "json";
+    recordSeparator: string;
```

Comment:
Is recordSeparator by design duplicated in child interfaces?

Reply:
Yes. As the parent interface shall not contain this field.